src/JuMP.jl

```diff
@@ -13,6 +13,9 @@ module JuMP
 using LinearAlgebra
 using SparseArrays
 
+import MutableArithmetics
+const MA = MutableArithmetics
```

Comment:
Should we be exporting `MA`? (Don't see why.)

Reply:
No indeed, I forgot we export everything that don't start with underscore